include/exadg/solvers_and_preconditioners/multigrid/multigrid_input_parameters.h

```diff
@@ -164,20 +165,30 @@ struct AMGData
 {
   AMGData()
   {
-    data.smoother_sweeps = 1;
-    data.n_cycles        = 1;
-    data.smoother_type   = "ILU";
+    trilinos_data.smoother_sweeps = 1;
+    trilinos_data.n_cycles        = 1;
+    trilinos_data.smoother_type   = "ILU";
+
+    boomer_data.n_sweeps_coarse = 1;
+    boomer_data.max_iter        = 1;
+    boomer_data.relaxation_type_down =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
+    boomer_data.relaxation_type_up =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
+    boomer_data.relaxation_type_coarse =
+      PETScWrappers::PreconditionBoomerAMG::AdditionalData::RelaxationType::Chebyshev;
   };
 
   void
   print(ConditionalOStream & pcout)
   {
-    print_parameter(pcout, "    Smoother sweeps", data.smoother_sweeps);
-    print_parameter(pcout, "    Number of cycles", data.n_cycles);
-    print_parameter(pcout, "    Smoother type", data.smoother_type);
+    print_parameter(pcout, "    Smoother sweeps", trilinos_data.smoother_sweeps);
+    print_parameter(pcout, "    Number of cycles", trilinos_data.n_cycles);
+    print_parameter(pcout, "    Smoother type", trilinos_data.smoother_type);
```

Comment:
How can I decide here which parameters to print?

Reply:
I would suggest that you add the BoomerAMG parameters to the print function with `if(false)` (so that the tests are passing) and I can later introduce an `enum AMGType {Trilinos, PETSc}` to implement the changes from #52 in a clean way (just `AMG` as coarse grid solver/preconditioner and an additional parameter `AMGType` specifying which AMG type is used; this way, the many if-statements in the code would be much more readable and less error-prone). Of course, I would be happy I you would like to address these changes already here (meaning adjusting several test cases and the tests).

Reply:
I introduced `enum AMGType` and the tests are running on my machine.